.travis.yml

```diff
@@ -18,13 +18,21 @@ matrix:
     - os: linux
       dist: trusty
       sudo: required
+    - os: linux
+      dist: trusty
+      sudo: required
+      env: SWIFT_SNAPSHOT=swift-4.2-DEVELOPMENT-SNAPSHOT-2018-07-01-a
     - os: osx
       osx_image: xcode9.2
       sudo: required
       env: SWIFT_SNAPSHOT=4.0.3
     - os: osx
       osx_image: xcode9.3
```

Comment:
Can you change this to xcode9.4 so that when this build runs, theres no Swift download (Xcode 9.4 comes with Swift 4.1.2).